modules/exploits/multi/http/joomla_http_header_rce.rb

```diff
@@ -0,0 +1,177 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = GoodRanking
```

Comment:
I think your module meets the requirements for ExcellentRanking?
https://github.com/rapid7/metasploit-framework/wiki/Exploit-Ranking